code/src/components/BackIcon.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const BackIcon = () => (
+  <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'>
```

Comment:
I think I need to come to you for some tips on using svgs like this. I'm sure I can google it, but I've got all this way with never actually doing it!